spec.html

```diff
@@ -27214,23 +27219,23 @@ <h1>Number.prototype.toExponential ( _fractionDigits_ )</h1>
         <emu-alg>
           1. Let _x_ be ? thisNumberValue(*this* value).
           1. Let _f_ be ? ToInteger(_fractionDigits_).
-          1. Assert: If _fractionDigits_ is *undefined*, then _f_ is 0.
+          1. Assert: If _fractionDigits_ is *undefined*, then _f_ is 0<sub>𝔽</sub>.
           1. If _x_ is not finite, return ! Number::toString(_x_).
-          1. If _f_ &lt; 0 or _f_ &gt; 100, throw a *RangeError* exception.
+          1. If _f_ &lt; 0<sub>𝔽</sub> or _f_ &gt; 100<sub>𝔽</sub>, throw a *RangeError* exception.
           1. Let _s_ be the empty String.
-          1. If _x_ &lt; 0, then
+          1. If _x_ &lt; 0<sub>𝔽</sub>, then
             1. Set _s_ to *"-"*.
             1. Set _x_ to -_x_.
           1. If _x_ = 0, then
             1. Let _m_ be the String value consisting of _f_ + 1 occurrences of the code unit 0x0030 (DIGIT ZERO).
             1. Let _e_ be 0.
           1. Else,
             1. If _fractionDigits_ is not *undefined*, then
-              1. Let _e_ and _n_ be integers such that 10<sup>_f_</sup> &le; _n_ &lt; 10<sup>_f_ + 1</sup> and for which ℝ(_n_) &times; 10<sub>ℝ</sub><sup>ℝ(_e_) - ℝ(_n_)</sup> - ℝ(_x_) is as close to zero as possible. If there are two such sets of _e_ and _n_, pick the _e_ and _n_ for which ℝ(_n_) &times; 10<sub>ℝ</sub><sup>ℝ(_e_) - ℝ(_f_)</sup> is larger.
+              1. Let _e_ and _n_ be integers such that 10<sup>ℝ(_f_)</sup> &le; _n_ &lt; 10<sup>ℝ(_f_) + 1</sup> and for which _n_ &times; 10<sup>_e_ - _n_</sup> - ℝ(_x_) is as close to zero as possible. If there are two such sets of _e_ and _n_, pick the _e_ and _n_ for which _n_ &times; 10<sup>_e_ - ℝ(_f_)</sup> is larger.
             1. Else,
-              1. Let _e_, _n_, and _f_ be integers such that _f_ &ge; 0, 10<sup>_f_</sup> &le; _n_ &lt; 10<sup>_f_ + 1</sup>, the Number value for ℝ(_n_) &times; 10<sub>ℝ</sub><sup>ℝ(_e_) - ℝ(_f_)</sup> is _x_, and _f_ is as small as possible. Note that the decimal representation of _n_ has _f_ + 1<sub>ℝ</sub> digits, _n_ is not divisible by 10, and the least significant digit of _n_ is not necessarily uniquely determined by these criteria.
+              1. Let _e_, _n_ be integers, and _f_ be a Number whose mathematical value is an integer such that ℝ(_f_) &ge; 0, 10<sup>ℝ(_f_)</sup> &le; _n_ &lt; 10<sup>ℝ(_f_) + 1</sup>, the Number value for _n_ &times; 10<sup>_e_ - ℝ(_f_)</sup> is ℝ(_x_), and _f_ is as small as possible. Note that the decimal representation of _n_ has ℝ(_f_) + 1 digits, _n_ is not divisible by 10, and the least significant digit of _n_ is not necessarily uniquely determined by these criteria.
```

Comment:
This is a good example of place where we could create a new variable to store MV of `x` and `f`. We have multiple calls of `ℝ(x)` and ` ℝ(f)`.